fe/fe-core/src/main/java/org/apache/doris/nereids/analyzer/UnboundRelation.java

```diff
@@ -42,34 +43,37 @@
  */
 public class UnboundRelation extends LogicalLeaf implements Relation, Unbound {
     private final List<String> nameParts;
+    private final RelationId id;
```

Comment:
it is better add 2 new abstract class: `AbstractLogicalRelation`, `AbstractPhysicalRelation` and put common attributes in them